### DISCUSSION TITLE
Lightroom Classic plugin for Immich Upload

### DISCUSSION THREAD

**Discussion by bmachek** [immich-app/immich#4561] [Show and tell]
Hi there,

I just created a Lightroom Classic Plugin, which enables to directly export images from LRC to your Immich server via Immich API.
The current state is alpha or proof of concept whatever you may call it. I got it to work just yet.
I pushed the source code to github. (https://github.com/bmachek/lrc-immich-plugin).

Anyone interested in testing or assisting me (I'm completely new to LUA) is very welcome.

Also ideas are very welcome.

I will do write some instructions how to use the plugin shortly.

Bastian

**Comment by rantanlan** [comment]
Just stumbled upon this and gave it a try, works well :) Thank you for that helpful addition :)
If I had to criticize anything it would be that on my end the fields for the addon are somewhat weirdly limited to one character on windows.
![image](https://github.com/immich-app/immich/assets/78840/49d5f3cd-01e3-4ec1-af37-bbb1354cdc40)

**Reply by bmachek** [reply]
Thank you for your feedback, it is appreciated. :-)

The input fields aren't supposed to be that small, and they aren't at my installation. Could you please try to resize the export dialog, to see if that changes the size of the input boxes? If not, please open an issue over at my repo...
